Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/local/git.py

```diff
@@ -26,7 +26,7 @@
 import re
 import time
 from collections import OrderedDict
-from datetime import datetime, timezone
+from datetime import datetime
```

Comment:
I think this is your only issue! If you re-add the `, timezone` import, I think all the tests will pass.

Reply:
I just did.